integration/tests/epochs/epoch_join_and_leave_vn_test.go

```diff
@@ -10,14 +10,26 @@ import (
 )
 
 func TestEpochJoinAndLeaveVN(t *testing.T) {
-	unittest.SkipUnless(t, unittest.TEST_FLAKY, "epochs join/leave tests should be run on an machine with adequate resources")
+	// TODO this test is blocked by https://github.com/dapperlabs/flow-go/issues/6443
+	unittest.SkipUnless(t, unittest.TEST_FLAKY, "this test is flaky due to an unhandled case in service event processing following epoch transition https://github.com/dapperlabs/flow-go/issues/6443")
```

Comment:
So you're keeping this test in quarantine?

Reply:
Yes this test will be quarantined until https://github.com/dapperlabs/flow-go/issues/6443 is fixed.